arcade/experimental/video_cv2.py

```diff
@@ -14,12 +14,16 @@
 import cv2  # type: ignore
 
 
-class CV2Player(arcade.Window):
-
-    def __init__(self):
-        super().__init__(800, 600, "OpenCV Video Player", resizable=True)
+class CV2Player(arcade.View):
```

Comment:
In a perfect world we should separate the player from the view. You don't have to do that, but it will surely make things more reusable.